admin/src/app/home/orders/stock-orders/fulfill/fulfill.component.html

```diff
@@ -1,4 +1,4 @@
-<ngx-loading [show]="loading"></ngx-loading>
+<ngx-loading [show]="loading > 0"></ngx-loading>
```

Comment:
`loading` is supposed to be a boolean, keep it so to maintain code consistency. I understand you keeping a counter to see how many times the loader is called, but it can be done in an easier manner where you just don't make the variable `false` until the line items are fetched. Anyway, see my feedback on not fetching all the line items after updating in case of bar code scanning